### DISCUSSION TITLE
FWI artefact for some profile

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#199] [Q&A] [answered]
Hi,

Shot point spacing is 200 m.
On the picture (clipped FWI result on 5 Hz) on the top you can see vertical lines (artefacts).
Can't unrestand why this is happening... any ideas?
![image](https://github.com/slimgroup/JUDI.jl/assets/43808863/28b8b4cd-87e3-4e06-aa0b-989da63e9431)

**Comment by mloubout** [comment, marked as answer]
Is this with `limit_m` ? If so you will get these type of artifacts if the aperture isn't large enough to have "zeros" at the edge especially for marine data so you may need larger buffer size

**Reply by kerim371** [reply]
`limit_m` is ON and the buffer size is 0.
But I used to do that on Viking Graben line 12 with 75 m shot spacing and there was such artefact (at least visually).
Here I have 8000m cable length... But I will test with larger buffer size and report here

**Comment by kerim371** [comment]
@mloubout you were right!
After increasing buffer size the probelm was solved.
Thank you!

**Comment by kerim371** [comment]
@mloubout I have one related question.

Do you have any ideas why such curvy lines at the top may appear?
They start to appear from 8Hz. 
At 5 Hz there no such artefacts..

The streamer length is 8km and and buffer_size=1.5 km
![image](https://github.com/slimgroup/JUDI.jl/assets/43808863/3964a8be-b581-4bb2-a3a8-ecca47340938)
